extensions.json

```
@@ -73,6 +73,11 @@
       "repository": "https://github.com/bmewburn/vscode-intelephense",
       "checkout": "v1.4.1"
     },
+    {
+      "id": "bradlc.vscode-tailwindcss",
+      "repository": "https://github.com/bradlc/vscode-tailwindcss"
+    },
+
```

I notice that this repository maintains consistent Git tags (e.g. `v0.3.1`) so I think it's better to publish this extension from the latest released Git tag, as opposed to straight from the `master` branch, using the `checkout` option. (Also, specifying `version` is an optimization that simply skips trying to re-publish the same version every night.)

```suggestion
    {
      "id": "bradlc.vscode-tailwindcss",
      "version": "0.3.1",
      "repository": "https://github.com/bradlc/vscode-tailwindcss",
      "checkout": "v0.3.1"
    },
```

---

Hi @jankeromnes, thanks for the clarification and the suggestion !

Maybe the README.md could be clearer on this subject, to me it looked like _completely_ optional values, now I understand that we _should_ specify checkout if version tags are available.